.github/workflows/build-with-bal-test-native-template..yml

```diff
@@ -0,0 +1,167 @@
+name: Build with bal test native
+
+on:
+    workflow_call:
+        inputs:
+            build_envs:
+                description: Test Environments
+                type: string
+                required: false
+                default: Ubuntu and Windows
```

Comment:
Do we need this? We do need to run in both environments, right? Is there a requirement to run either one separately?

We might be able to define the two jobs separately and create a composite workflow as well if that's a concern.

Reply:
I have added this because currently `bal test --native` is failing for windows. And the fix is done now but not yet merged. 
So I don't think there is a requirement for this now, will remove this

Reply:
Removed via [0203dfb](https://github.com/ballerina-platform/ballerina-standard-library/pull/3615/commits/0203dfbcd9125459576ad90f502d5e9c2dcd6e49)